examples/Coinbase/Spot.jl

```diff
@@ -14,4 +14,8 @@ CryptoAPIs.Coinbase.Spot.candle(;
 
 CryptoAPIs.Coinbase.Spot.currency()
 
+CryptoAPIs.Coinbase.Spot.fee_estimate()
```

Comment:
Since the method is private, it is necessary to declare the client in advance and pass it as an argument to the `free_estimate()` method